lessons/05_relax/03b_ConjugateGradient.ipynb

```diff
@@ -0,0 +1,541 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "###### Content under Creative Commons Attribution license CC-BY 4.0, code under MIT license (c)2014 L.A. Barba, C.D. Cooper, G.F. Forsyth.  "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Let's head in the right direction!"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In the previous notebook we applied the Jacobi method to the resolution of the Poisson equation and you also learned earlier how several other methods and techniques could fasten the iterative resolution of elliptic PDEs. However, there is still room for improvement. Before looking at the multigrid technique, let us consider the very popular conjugate gradient (CG) method and apply it to the Poisson equation problem that we solved in Lesson 2. The CG method is either used on its own, or in conjunction to multigrid where the latter is used as a preconditionner (more on that in lesson XX)."
```

Comment:
to fasten is to attach, like a button -- perhaps

"could fasten" -> "can speed up"

and no double 'n' at the end of preconditionner